res/xml/bone.xml

```diff
@@ -0,0 +1,45 @@
+<?xml	version="1.0"	encoding="utf-8"?>
+<keyboard bottom_row="false">
+	<row>
+		<key key0="j"				key2="°"				key4="esc"/>
+		<key key0="d"				key2="§"				key4="_"/>
+		<key key0="u"				key2="ℓ"				key4="["/>
+		<key key0="a"				key2="↻"				key4="]"/>
+		<key key0="x"				key2="↺"				key4="^"/>
+		<key key0="p"	key1="$"				key3="!"	key4="7"/>
+		<key key0="h"	key1="€"				key3="&lt;"	key4="8"/>
+		<key key0="l"	key1="£"				key3="&gt;"	key4="9"/>
+		<key key0="m"	key1="¥"				key3="="	key4="f11_placeholder"/>
+		<key key0="w"	key1="¢"				key3="&amp;"	key4="f12_placeholder"/>
```

Comment:
The currency symbols are in the `Fn` layer already, no need to put them here.

Reply:
removed